addons/zylann.hterrain/hterrain_data.gd

```diff
@@ -1349,18 +1349,23 @@ func _import_heightmap(fpath: String, min_y: float, max_y: float, big_endian: bo
 		convert_float_heightmap_to_rgb8(src_image, im)
 
 	elif ext == "raw":
-		# RAW files don't contain size, so we have to deduce it from 16-bit size.
+		# RAW files don't contain size, so we have to deduce it from 16-bit/24-bit size.
 		# We also need to bring it back to float in the wanted range.
 
 		var f := FileAccess.open(fpath, FileAccess.READ)
 		if f == null:
 			return false
 
+		var is_24_bit : bool
 		var file_len := f.get_length()
-		var file_res := HT_Util.integer_square_root(file_len / 2)
+		var file_res := HT_Util.integer_square_root(file_len / 2) # test 16-bit
 		if file_res == -1:
-			# Can't deduce size
-			return false
+			file_res = HT_Util.integer_square_root(file_len / 3) # test 24-bit
```

Comment:
I'm not sure this is a good idea... we should better know up-front that the file is 24-bit, because we could accidentally find a matching 16-bit resolution even though the file is 24-bit. It should be part of options (RAW 16 bit, RAW 24 bits)

Reply:
Done. Most recent commit added OptionButton to select bit depth